.storybook/preview.js

```diff
@@ -9,12 +9,13 @@ import {
   parseBrandingJson,
 } from '@/utils/parseBrandingJson';
 import { injectThemeCssVariables } from '@/utils/injectThemeCssVariables';
-import { getThemeNames } from './utils/getThemeNames';
 import { getTheme } from './utils/getTheme';
+import { getThemeNames } from './utils/getThemeNames';
 
 const isExternal = process.env.STORYBOOK_THEME_LOCATION === 'external';
+const isFigma = process.env.STORYBOOK_THEME_LOCATION === 'figma';
```

Comment:
do we need this env var in the sample?

Reply:
nope. it gets set in the buildscripts. Its not one you have to set in `.env`. you can just run the scripts `sb:external` or `sb:figma`